tox.ini

```diff
@@ -15,20 +15,15 @@ deps =
     pep257
```

Comment:
I cleaned this file up a bit to make it easier to call the linters without calling tox.